### PR TITLE
Add missing cmath for cos(...) usage

### PR DESCRIPTION
**Pull request by mbargull**
Hi there,

Over at https://github.com/bioconda/bioconda-recipes/ we had a linking issue while updating our rDock package to version `0.22`.
Our build linked `libRbt` via `RbtDihedralSF.o` with the mangled `_ZGVbN2v_cos` symbol which is available from `libvecm` from glibc >=2.22 .
To create compatible binaries, we link with older glibc versions and as such our build failed due to the missing library/symbol.
Adding the `#include <cmath>` gives us the desired linking with the `(GLIBC_2.2.5) cos` symbol.

Cheers,
Marcel
cc @simonbray

### CODE REVIEW

File: src/lib/RbtDihedralSF.cxx [hunk 1]
```diff
@@ -11,6 +11,7 @@
  ***********************************************************************/
 
 #include "RbtDihedralSF.h"
+#include <cmath>
```
Comment (ggutierrez-sunbright) [inline src/lib/RbtDihedralSF.cxx:15]:
```suggestion
#include <cmath>

#include "RbtDihedralSF.h"

```

Comment (mbargull) [reply src/lib/RbtDihedralSF.cxx:15]: Feel free to directly push any necessary changes to this branch.
("Allow edits by maintainers" is checked.)

**Comment by ggutierrez-sunbright** [comment]
nevermind about the include reordering, just realized the `"RbtDihedralSF.h" is the main header and that's why it goes on top :expressionless:

**Review by ggutierrez-sunbright — APPROVED** [review]

**Comment by mbargull** [comment]
> nevermind about the include reordering, just realized the `"RbtDihedralSF.h" is the main header and that's why it goes on top 😑

No worries -- I couldn't remember the details on this anyway :D.

**Comment by ggutierrez-sunbright** [comment]
I'm merging this now, but about this:
> Over at https://github.com/bioconda/bioconda-recipes/ we had a linking issue while updating our rDock package to version `0.22`.

can we contribute in any way to help on this?
the new Makefile should make things much easier, and there's pre-built bundles as release assets for a variety of distributions coming from our CI/CD pipeline now

I created this issue #60 to keep the conversation over there

thanks and regards, and sorry for the delay

**Pull request merged**